src/assets/icon.svg

```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="UTF-8"?><svg viewBox="0 0 1000 950" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="0" gradientTransform="rotate(90)"><stop stop-color="#D73F4A"/><stop offset="1" stop-color="#C22B36"/></linearGradient><linearGradient id="1" gradientTransform="rotate(90)"><stop stop-color="#3FCD46"/><stop offset="1" stop-color="#27BB2E"/></linearGradient><linearGradient id="2" gradientTransform="rotate(90)"><stop stop-color="#3F5CD7"/><stop offset="1" stop-color="#2B48C2"/></linearGradient><filter id="3"><feOffset dy="10"/><feGaussianBlur result="b" stdDeviation="20"/><feFlood flood-opacity=".8"/><feComposite in2="b" operator="in"/><feComposite in="SourceGraphic"/></filter><path id="4" d="M320.733,150.5 C395.427,21.125 604.573,21.125 679.267,150.5 L918.29,564.5 C992.985,693.875 888.412,875 739.023,875 L260.977,875 C111.588,875 7.015,693.875 81.71,564.5 Z"/></defs><use href="#4" fill="#404040"/><path d="M800.52,350.104 L620.208,169.792 L379.792,410.208 L620.208,650.624 L592.674,710.104 L412.362,529.792 L171.946,770.208 L280.133,878.395 L739.023,875 C888.412,875 992.985,693.875 918.29,564.5 Z" fill-opacity=".125"/><circle cx="500" cy="290" r="170" fill="url(#0)"/><circle cx="707.846" cy="650" r="170" fill="url(#1)"/><circle cx="292.154" cy="650" r="170" fill="url(#2)"/><use href="#4" fill="none" stroke="#fff" stroke-width="40" filter="url(#3)"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?><svg viewBox="0 0 1000 950" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="r" gradientTransform="rotate(90)"><stop stop-color="#D73F4A"/><stop offset="1" stop-color="#C22B36"/></linearGradient><linearGradient id="g" gradientTransform="rotate(90)"><stop stop-color="#3FCD46"/><stop offset="1" stop-color="#27BB2E"/></linearGradient><linearGradient id="b" gradientTransform="rotate(90)"><stop stop-color="#3F5CD7"/><stop offset="1" stop-color="#2B48C2"/></linearGradient><filter id="s"><feOffset dy="10"/><feGaussianBlur result="b" stdDeviation="20"/><feFlood flood-opacity=".8"/><feComposite in2="b" operator="in"/><feComposite in="SourceGraphic"/></filter><path id="t" d="M320.733,150.5C395.427,21.125 604.573,21.125 679.267,150.5L918.29,564.5C992.985,693.875 888.412,875 739.023,875L260.977,875C111.588,875 7.015,693.875 81.71,564.5Z"/></defs><use href="#t" fill="#404040"/><path d="M786.297,335.88L620.208,169.792L379.792,410.208L620.208,650.624L592.674,710.104L412.362,529.792L171.946,770.208L276.738,875L739.023,875C888.412,875 992.985,693.875 918.29,564.5Z" fill-opacity=".125"/><circle cx="500" cy="290" r="170" fill="url(#r)"/><circle cx="707.846" cy="650" r="170" fill="url(#g)"/><circle cx="292.154" cy="650" r="170" fill="url(#b)"/><use href="#t" fill="none" stroke="#FFF" stroke-width="40" filter="url(#s)"/></svg>
```

Comment:
What changed here?

Reply:
Not much, it's just that the previous master didn't use the final version that we ended up with, so I updated it to that. I believe it should match all the other repositories already.